backend/box/box.go

```diff
@@ -293,6 +294,7 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 			}
 			return nil, err
 		}
+		f.features.Fill(&newF)
```

Comment:
What is this change for?  Hasn't features been fill-ed already?

Reply:
In the case the case that root is a file a copy of the fs struct is created and used but operations like public link take the function pointers from the Feature list which still use the old object. I wasn't sure if this was intentional or not. In my own manual test share TestBox:Testfile f.Root as accessed from PublicLink will be "Testfile" which would make the Parameter to PublicLink redundant.

Reply:
Oh - good catch.  I think I've made that mistake before and likely it is in other backends too...